src/platform/Infineon/CYW30739/ConfigurationManagerImpl.cpp

```diff
@@ -51,9 +52,11 @@ CHIP_ERROR ConfigurationManagerImpl::Init()
     {
         err = GetRebootCount(rebootCount);
         SuccessOrExit(err);
-
-        err = StoreRebootCount(rebootCount + 1);
-        SuccessOrExit(err);
+        if (wiced_sleep_get_boot_mode() != WICED_SLEEP_FAST_BOOT)
```

Comment:
Another question: when is `PlatformMgrImpl().GetStartTime()` being set, for purposes of GetUpTime?  That one:

1) Needs to match the reboot count bit
2) Needs to match the time base for events.

Reply:
For the first question, we will save data into retention RAM before sleep. And it count as "wakes up from a sleep state".
For the second question, I think PlatformMgrImpl().GetStartTime() is for Linux specific, and we didn't implement it on our platform.

Reply:
@AlvinHsiao Which data is being saved into retention RAM?

I see that I just misread things and GetStartTime is relevant to `Infineon/PSOC6`, not `Infineon/CYW30739`, and that `Infineon/CYW30739` just does not implement the UpTime attribute at all.  Is that a correct description of the current situation?

And your're shutting down the Matter stack before entering the sleep state and restarting it when you exit?

Reply:
@bzbarsky-apple Yes, this is our temporary workaround to support sleepy end device.

Reply:
@HuangCC-IFX That doesn't quite answer my question around which data is persisted.  For example, when you enter this sleep state, do you clear out the queued events in the Matter stack?  Do you issue a new StartUp event when coming out and a ShutDown event when entering?